src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1227,7 +1267,7 @@ private void SetBorder(Rectangle value)
         /// Lazy call to UpdateImage.
         /// Collect Properties need to be update, and set properties that starts the Processing.
         /// </summary>
-        private void UpdateImage(int key, PropertyValue value)
+        internal protected void UpdateImage(int key, PropertyValue value)
```

Comment:
This "protected internal" means internal or protected.
This method can be used in the inherit class of ImageView even thought it is in app.
Is it right you wanted?

Reply:
Thanks for notify that. I want to use this API as internal AND protected.
And this case, the right usage keyword was 'protected internal'. not 'internal protected'.

I change some codes as protected internal level so LottieAnimationView can call that API.

Reply:
But if it is changed to protected internal, it can be opened as public.
Is it right what you wanted?

Reply:
ohno... `protected internal` was `protected` OR `internal` -_-;; That was my mistake. Fix done.